doc/source/whatsnew/v0.18.0.txt

```diff
@@ -1117,3 +1117,4 @@ Bug Fixes
 - Bug in ``DataFrame.apply`` in which reduction was not being prevented for cases in which ``dtype`` was not a numpy dtype (:issue:`12244`)
 - Bug when initializing categorical series with a scalar value. (:issue:`12336`)
 - Bug when specifying a UTC ``DatetimeIndex`` by setting ``utc=True`` in ``.to_datetime`` (:issue:11934)
+- Bug in ``searchsorted`` argument signatures for ``IndexOpsMixin``, ``DatetimeIndex``, ``PeriodIndex``, and ```TimedeltaIndex`` that broke compatibility with ``searchsorted`` in ``numpy`` (:issue:`12238`)
```

Comment:
you don't need to say all this, just say ``searchsorted`` now accepts ``sorter`` arg for compat with numpy.